helpers/exports.js

```diff
@@ -0,0 +1,28 @@
+import { User } from '../models';
+import sendVerificationEmail from './sendmail';
+
+exports.resendVerificationEmail = async (req, res) => {
+  const { email } = req.body.user;
+  try {
+    const usertoBeVerified = await User.findOne({
+      where: { email }
+    });
+    if (usertoBeVerified === null || usertoBeVerified === 'null' || usertoBeVerified.isverified === true) {
+      return res.status(400).json({
+        errors: {
+          body: ['Your account has already been verified or the email address has not been registered']
```

Comment:
Please separate these two. Account verified should be separated from email registered.